### PR TITLE
Added Plebstyle Titan Wallet and Terrasteel Wallet

### PR DESCRIPTION
**Pull request by Plebstyle**
Added 2 new json files for the two seed backups.
What else do you need? Pictures?

I will sent you the Code and Ref-Link on Nostr

BR,
Jonathan

### CODE REVIEW

File: items/plebstyle-terrasteel-wallet.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
+{
+  "id": "Plebstyle-Titan-Wallet",
```
Comment (maxirosson) [inline items/plebstyle-terrasteel-wallet.json:2]: Id should be in lowercase.  "plebstyle-titan-wallet"

File: items/plebstyle-terrasteel-wallet.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      "value": "YES",
+      "supported": true
+    }
+  },
+  "size-materials": {
+    "weight": {
+      "value": "250g / 8.82oz"
+    },
+    "dimensions": {
+      "value": "65mm x Ø27mm (Caps: Ø33mm)"
```
Comment (maxirosson) [inline items/plebstyle-terrasteel-wallet.json:65]: For consistency, lets do this:

```
"value": "65 x 27 x 27mm",
"texts": [
    "Caps: Ø33mm"
]
```

File: items/plebstyle-terrasteel-wallet.json [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    },
+    "materials": {
+      "value": "Stainless Steel",
+      "texts": [
+        "316/316L Stainless Steel"
+      ]
+    },
+    "include-all-tools": {
+      "value": "Start Set required or Print@Home",
```
Comment (maxirosson) [inline items/plebstyle-terrasteel-wallet.json:74]: Let's do this for consistency:

```
"value": "Start Set required"
"links": [
  {
    "title": "Print@Home",
    "url": "https://www.thingiverse.com/plebstyle/designs"
  }
],
"flag": "neutral",
"supported": true
```

File: items/plebstyle-terrasteel-wallet.json [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      "flag": "negative",
+      "supported": false
+    },
+    "encrypted-backup": {
+      "value": "NO",
+      "flag": "negative",
+      "supported": false
+    },
+    "tamper-evident-seal": {
+      "value": "Add. Option",
```
Comment (maxirosson) [inline items/plebstyle-terrasteel-wallet.json:131]: What this means?

File: items/plebstyle-titan-wallet.json [hunk 5]
```diff
@@ -0,0 +1,180 @@
+{
+  "id": "Plebstyle-Titan-Wallet",
```
Comment (maxirosson) [inline items/plebstyle-titan-wallet.json:2]: Id should be in lowercase.

File: items/plebstyle-titan-wallet.json [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+      "value": "YES",
+      "supported": true
+    }
+  },
+  "size-materials": {
+    "weight": {
+      "value": "420g / 14.82oz"
+    },
+    "dimensions": {
+      "value": "75mm x Ø27mm (Caps: Ø33mm)"
```
Comment (maxirosson) [inline items/plebstyle-titan-wallet.json:65]: IDEM

File: items/plebstyle-titan-wallet.json [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+    },
+    "materials": {
+      "value": "Stainless Steel and Titanium",
+      "texts": [
+        "Titanium alloyed Stainless Steel 316Ti and Titanium Grade 5 – Perfect for SIMULTANEOUS strength and heat resistance"
+      ]
+    },
+    "include-all-tools": {
+      "value": "Start Set required or Print@Home",
```
Comment (maxirosson) [inline items/plebstyle-titan-wallet.json:74]: IDEM

**Review by maxirosson — COMMENTED** [review]
Thanks so much for your collaboration. I added some comments to the JSONs

File: items/plebstyle-terrasteel-wallet.json (hunk 4 above)
Comment (Plebstyle) [reply items/plebstyle-terrasteel-wallet.json:131]: The tamper-evident-seal is not included, but you can buy one in the store (5 for 3€)

**Pull request merged**